x-pack/plugins/security_solution/cypress/integration/alerts_detection_exceptions.spec.ts

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { exception } from '../objects/exception';
+
+import { RULE_STATUS } from '../screens/create_new_rule';
+import { SERVER_SIDE_EVENT_COUNT } from '../screens/timeline';
+
+import {
+  addExceptionFromFirstAlert,
+  goToClosedAlerts,
+  goToManageAlertsDetectionRules,
+  goToOpenedAlerts,
+  waitForAlertsIndexToBeCreated,
+} from '../tasks/alerts';
+import { goToRuleDetails } from '../tasks/alerts_detection_rules';
+import { waitForAlertsToPopulate } from '../tasks/create_new_rule';
+import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
+import {
+  activatesRule,
+  addsException,
+  addsExceptionFromRuleSettings,
+  deactivatesRule,
+  goToAlertsTab,
+  goToExceptionsTab,
+  removeException,
+  waitForTheRuleToBeExecuted,
+} from '../tasks/rule_details';
+import { refreshPage } from '../tasks/security_header';
+
+import { DETECTIONS_URL } from '../urls/navigation';
+
+const numberOfauditbeatExceptionsAlerts = 2;
```

Comment:
nit - could we rename this to `numberOfAuditbeatExceptionAlerts` to follow camelCase?